tests/tracer/test_encoders.py

```diff
@@ -174,6 +174,12 @@ def test_encode_traces_json(self):
                 Span(name="client.testing", tracer=None),
             ]
         )
+        traces.append(
```

Comment:
We could initialize traces with all the spans instead of appending them in chunks.
```suggestion
        traces = [
            [ 
                Span(name="client.testing", tracer=None),
                Span(name="client.testing", tracer=None),
            ],
            [
               Span(name="client.testing", tracer=None),
               Span(name="client.testing", tracer=None),
            ],
           [
               Span(name=b"client.testing", tracer=None),
               Span(name=b"client.testing", tracer=None),
           ],
        ]
```

Reply:
yes we could 👍🏻